reports/docusign-autopopulation/index.md

```diff
@@ -0,0 +1,134 @@
+ 
+
+### Skuid Configure: 
+1.	Go to Data Sources > Authentication Providers
+2.	Create a new Authentication Provider with a unique name and DocuSign Provider type. 
+3.	Set the Authorize Endpoint URL and Token Endpoint URL to: 
+	- Authorize: `https://account-d.docusign.com/oauth/auth`
+	-	Token: `https://account-d.docusign.com/oauth/token`
+4.	Set the Client Id to the Integrator Key Id defined in DocuSign
+5.	Set the Client Secret to the Secret Key you saved from DocuSign earlier. Save!
+6.	Now go to Data Sources > Data Sources
+7.	Create a New Data Source with a unique name and with the REST Data Source Type
+8.	Set the URL to  `https://demo.docusign.net/restapi` and select the Authentication Provider that you created in Steps 1-5.
+
+
+### Skuid Page:
+1.	Create a Model with REST as the data source type, and select the DocuSign data source that you created. Select Read/Write as the Model Behavior.
+2.	On this new model, select Methods>insert
+3.	Set values as follows: 
+	-	type = Insert
+	-	url = `/accounts/{{AccountId}}/envelopes/`
+	- HTTP verb = POST 
+	- Send new field values = “As templated request body” 
+4.	On this model, create a field for every piece of data that you want to insert into the DocuSign template. Also include fields for undefined recipients (name and email). 
+5.	For each field you create, set the default value to point to the data you want to merge (Field from another model is the easiest way to do this).
+6.	Get ready to create your request body!
+
+
+### Creating Your Templated Request Body:
+Because DocuSign’s REST API is fairly complex, creating a templated request body with merge variables is the way to go. (View an example below!)
+You can basically follow the format given in the [DocuSign API Explorer](https://apiexplorer.docusign.com/#/esign/restapi?categories=Envelopes&tags=Envelopes&operations=create) but you will only need the following fields: 
+1.	Envelope
+	- status - either sent or created (for a draft)
+	- templateId - referenced above. 
+	- emailSubject - define the subject of the email to be sent with the DocuSign document. 
+2.	Recipient
+	- name
+	- email
+	- roleName - same as defined in the DocuSign template
+	- recipientId - the best way to get this is to do a [get request for recipients](https://apiexplorer.docusign.com/#/esign/restapi?categories=Envelopes&tags=EnvelopeRecipients&operations=list) on the api explorer 
+	- routingOrder (optional) - can also be defined in DocuSign, but it can be helpful to conditionally choose the routing order in Skuid. 
+3.	Recipient tab
+	- tabLabel - Data Label defined in DocuSign
+	- value - the value you want to insert (use merge syntax here!)
+	- font (optional) - choose a font to stylize your text. (Supported Fonts: Arial, Arial, ArialNarrow, Calibri, CourierNew, Garamond, Georgia, Helvetica,   LucidaConsole, Tahoma, TimesNewRoman, Trebuchet, Verdana, MSGothic, MSMincho, Default)
+
+#### Example Request Body:  
+ 
```

Comment:
I know it's not much, but let's get some [Markdown syntax highlight](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code) here:
```suggestion
```json
```